dip-template/nodes/dip-consumer/src/chain_spec.rs

```diff
@@ -16,6 +16,10 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
+// Triggered by the `ChainSpecGroup` derive macro used for the custom chainspec
+// extension.
+#![allow(clippy::derive_partial_eq_without_eq)]
```

Comment:
the reason, why it is allowed, should maybe go into the allow. We can also enforce that with: https://rust-lang.github.io/rust-clippy/master/index.html#allow_attributes_without_reason

Reply:
Unfortunately allow and expect reasons were still unstable in 1.74, because they were only stabilised in 1.81, hence we cannot use them here. I added two more lints that can be uncommented if and when we upgrade our toolchain to 1.81: https://github.com/KILTprotocol/kilt-node/pull/762/commits/7b2ef121459d012a3ccdb084ea4f5808d489cab4.